### ISSUE TITLE
Doesn't works

### ISSUE THREAD

**Issue by Kimapr** [temtemtemidoit/python-code-to-crash-your-pc#1]
sh: 1: start: not found
    sh: 1: start: not found
    sh: 1: start: not found
    sh: 1: start: not found
    sh: 1: start: not found
    sh: 1: start: not found
    sh: 1: start: not found
    ...

**Comment by jordan4ibanez** [comment]
Yeah I tried this code, but unfortunately I don't even have a PC so it really didn't work

**Comment by temtemtemidoit** [comment]
kimapr are you on linux?

**Comment by temtemtemidoit** [comment]
fixed

**Comment by Kimapr** [comment]
not fixed

**Comment by jordan4ibanez** [comment]
It's unfixed :(
